src/Mantasflowers.Services/ServiceAgents/Exceptions/FirebaseTokenException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Mantasflowers.Services.ServiceAgents.Exceptions
+{
+    [Serializable]
+    public class FirebaseTokenException : Exception
+    {
+        public FirebaseTokenException() { }
+        public FirebaseTokenException(string message) : base(message) { }
+        public FirebaseTokenException(string message, Exception inner) : base(message, inner) { }
```

Comment:
What's the use case for inner exception? Could it actually be FirebaseAdmin.FirebaseException?

Reply:
Its just a template vscode auto generates for custom exceptions